app/Libraries/Payments/PaypalCreatePayment.php

```diff
@@ -92,7 +102,7 @@ private function getAmount()
 
     private function getItemList()
     {
-        return (new ItemList())
+        $itemList = (new ItemList())
```

Comment:
Check `requiresShipping` in `getShippingAddress` (returning `null` accordingly) instead so it can still be just `return new ItemList(...)`?